### PR TITLE
Update init script to append STDERR to log.

### PR DESCRIPTION
**Pull request by benbjohnson**
## Overview

This pull request changes `init.sh` to append standard error to the existing log, if one exists.

### CODE REVIEW

**Comment by benbjohnson** [comment]
@otoolep

**Comment by otoolep** [comment]
Shouldn't we do the same for `STDOUT`? It, too, could be a file.

**Comment by otoolep** [comment]
Update the CHANGELOG. Once you answer (and possibly address) the question about `STDOUT` +1.

**Comment by benbjohnson** [comment]
I wasn't sure what would happen if I tried to append to `/dev/null` so I figured I'd just do `STDERR` for now. I should have just tried it out. It's fine. I added it & updated the `CHANGELOG`.

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,7 @@
 - [#2151](https://github.com/influxdb/influxdb/pull/2151): Ignore replay commands on the metastore.
 - [#2156](https://github.com/influxdb/influxdb/pull/2156): Propagate error when resolving UDP address in Graphite UDP server.
 - [#2163](https://github.com/influxdb/influxdb/pull/2163): Fix up paths for default data and run storage.
+- [#2164](https://github.com/influxdb/influxdb/pull/2164): Append STDOUT/STDERR in initscript.
```
Comment (otoolep) [inline CHANGELOG.md]: Super nit-pick: typo? initscript -> init script

**Pull request merged**